dataset/test/bin_test.cpp

```diff
@@ -89,6 +90,16 @@ TEST_F(DataArrayBinTest, 2d) {
   EXPECT_EQ(bin(bin(table, {edges_x}), {edges_y}), bucketed);
 }
 
+TEST_F(DataArrayBinTest, operations_on_empty) {
+  const Variable indices = makeVariable<std::pair<scipp::index, scipp::index>>(
+      Dimensions{{Dim::Event, 0}, {Dim::Y, 0}}, Values{});
+  const Variable binned = make_bins(indices, Dim::Event, data);
+
+  EXPECT_EQ(abs(binned), binned);
+  EXPECT_EQ(binned, binned * binned);
+  EXPECT_EQ(binned, binned * (2 * units::one));
```

Comment:
Not calling `bin`, is this intentional? If so, this test should go into `bins_test.cpp`?

Reply:
Intentional because the version with `bin` is more complex than necessary to track this bug.
-> Moved